libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -137,22 +137,31 @@ void GLSLPostProcessor::spirvToToMsl(const SpirvBlob *spirv, std::string *outMsl
 
     auto executionModel = mslCompiler.get_execution_model();
 
-    auto duplicateResourceBinding = [executionModel, &mslCompiler](const auto& resource) {
+    auto duplicateResourceBinding = [executionModel, &mslCompiler](const auto& resource, uint16_t index = UINT16_MAX) {
```

Comment:
comment here explaining the parameters (e.g. what's `index`?)

Reply:
Ok it's done.